Kickstarter-iOS/Views/Cells/PledgeAmountCell.swift

```diff
@@ -80,6 +88,19 @@ final class PledgeAmountCell: UITableViewCell, ValueCell {
 
     self.amountInputView.label.rac.text = self.viewModel.outputs.currency
     self.amountInputView.textField.rac.text = self.viewModel.outputs.amount
+
+    self.viewModel.outputs.amountPrimitive
+      .observeForUI()
+      .observeValues { [weak self] amount in
+        guard let self = self else { return }
+        self.delegate?.pledgeAmountCell(self, didUpdateAmount: amount)
+      }
+  }
+
+  // MARK: - Actions
+
+  @objc func amountUpdated(_ textField: UITextField) {
+    self.viewModel.inputs.amountUpdated(to: textField.text ?? "")
```

Comment:
Should we just send through the `textField.text` regardless of whether it's `nil` or not and keep that logic all in the view model?

Reply:
Yip, I think I might've done this to test 👍

Reply:
This change is already made in #719... it may be better to try and merge that first, that way the stepper will also update the total.